homework-1/src/templates/tips.html

```diff
@@ -0,0 +1,107 @@
+<div class="tips" id="tips">  
+    <div class="tips__section">
+     <div class="tips__area-logo">
+        <a id="tipsLogo" class="tips__logo-link"><img src="/assets/images/SkoutTips.png" alt="" class="tips__logo" ></a>
+     </div>  
+     <div class="tips__first" id="tipsFirstOne">
+        <p class="tips__first-one-letter">1</p>
+        <p class="tips__first-text">System Patching</p>
+        <div class="tips__area-icon-plus" id="tipsPlusOne">
+            <p class="tips__icon-plus">+</p>
+        </div>
+     </div>
+     <div class="tips__second" id="tipsSecondOne">
```

Comment:
You have to show-hide content only, not whole tip with header.

You can find simple accordion ([for example this](https://www.w3schools.com/howto/howto_js_accordion.asp)) - see how it works and try to adapt it for yourself

Reply:
done